menudata.js

```diff
@@ -21,5 +21,8 @@ export default {
         past24Hours:            'Accessed in the Past 24 H&ours',
         today:                  'Accessed &Today',
         yesterday:              'Accessed &Yesterday',
+    },
+    'Clear': {
+        clearSelection:         'Clear selectio&n',
```

Comment:
```suggestion
    'Other': {
        none:         'Clear selection (&Z)',
```
Note that the commands are named to reflect their outcomes. So I prefer to call it `none`.

I don't feel `N` is right for this command. `C` is already taken. `Z` seems appropriate, it reminds us of 'undo'. Your hydrate function will need to remove `(&?)` as well.